source/app.js

```diff
@@ -0,0 +1,64 @@
+import browser from 'detect-browser'
+import translations from './translations'
+
+(function () {
+  let options = INSTALL_OPTIONS
+  const {localStorage = {}} = window
+  const DAY_DURATION = 1000 * 60 * 60 * 24
+  const now = new Date()
+  const weekAgo = new Date(now - DAY_DURATION * 7)
+
+  const language = window.navigator.language || window.navigator.userLanguage || 'en'
+  const [messageLabel, moreLabel] = translations[language] || translations[language.substring(0, 2)] || translations.en
+
+  const appElement = document.createElement('cloudflare-app')

```

Comment:
This and other element stuff could be moved after line 42 (where `return` is done) to avoid unnecessary calculations if the warning isn't going to be shown either way. Same goes for the language and translation variables.